pkg/controller/nodelifecycle/config/types.go

```diff
@@ -46,5 +46,5 @@ type NodeLifecycleControllerConfiguration struct {
 	// unhealthyZoneThreshold (no less than 3) of Nodes in the zone are NotReady
 	UnhealthyZoneThreshold float32
 	// tenant api-server URLs
-	TenantServers []string
+	Kubeconfigs []string
```

Comment:
Let's rename it to TenantPartitionKubeConfigs or sth. similar

Reply:
fixed